common/debugScreen_custom.h

```diff
@@ -0,0 +1,17 @@
+#ifndef DEBUG_SCREEN_CUSTOM_H
+#define DEBUG_SCREEN_CUSTOM_H
+
+//#define SCREEN_TAB_SIZE (8)
+
+// backward compatibility for sources based on older Vita SDK versions
+#define DEBUG_SCREEN_CODE_INCLUDE // not recommended for your own projects, but for sake of backward compatibility
```

Comment:
> not recommended for your own projects, but for sake of backward compatibility

yet you use them in the demo sample.
Why not `#define` it just in the sample that need them ?